### PR TITLE
Combine dependency updates 31 Aug 2023

### PR DESCRIPTION
**Pull request by mlilynolting**
Includes config change to make scala-steward run once weekly.

workbench-util2 upgraded to 0.7.

**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

[cloned from https://github.com/broadinstitute/workbench-libs/pull/1507]

### CODE REVIEW

File: .scala-steward.conf [hunk 1]
```diff
@@ -25,9 +25,10 @@
 #     the given expression.
 #
 # Default: @asap
-#
-#pullRequests.frequency = "0 0 ? * 3" # every thursday on midnight
-#pullRequests.frequency = "15 days"
+# Examples:
+#   pullRequests.frequency = "0 0 ? * 3" # every thursday on midnight
+#   pullRequests.frequency = "15 days"
+pullRequests.frequency = "0 0 ? * 1" # At 12:00 AM, only on Monday
```
Comment (LizBaldo) [inline .scala-steward.conf:31]: Thank you!

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Workbench utility libraries, built for 2.13. You can find the full list of packa
 Contains utility functions and classes. Util2 is added because util needs to support 2.11 for `firecloud-orchestration`,
 but many libraries start to drop 2.11 support. Util2 doesn't support 2.11.
 
-Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-util2" % "0.5-026bc90"`
+Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-util2" % "0.7-TRAVIS-REPLACE-ME"`
```
Comment (LizBaldo) [inline README.md:39]: Why bump by 2 here?

Comment (mlilynolting) [reply README.md:39]: The log was out of date. 0.6 existed already; see `util2/CHANGELOG.md`

Comment (LizBaldo) [reply README.md:39]: Ah good catch, thanks!

**Review by LizBaldo — APPROVED** [review]

**Pull request merged**